addons/airway/functions/fnc_updateInjuryList.sqf

```diff
@@ -162,6 +162,30 @@ if (_target getVariable [QGVAR(recovery), false]) then {
     _entries pushback [LLSTRING(RecoveryPosition), [0.1, 1, 1, 1]];
 };
 
+// Display cyanosis in overview tab, only when head/arms are selected
+if (EGVAR(breathing,cyanosisShowInMenu) && (_selectionN isEqualTo 0 || _selectionN isEqualTo 2 || _selectionN isEqualTo 3)) then {
+	private _spO2 = 0;
+	
+	if (alive _target) then {
+        _spO2 = GET_SPO2(_target);
+	};
+	
+    if (_spO2 <= EGVAR(breathing,slightValue)) then {
+        private _cyanosisArr = switch (true) do {
+            case (_spO2 <= EGVAR(breathing,severeValue)): {
+                ["STR_kat_breathing_CyanosisStatus_Severe",[0.16, 0.16, 1, 1]];
+            };
+            case (_spO2 <= EGVAR(breathing,mildValue)): {
+                ["STR_kat_breathing_CyanosisStatus_Mild",[0.16, 0.315, 1, 1]];
```

Comment:
use string macros

Reply:
With LELSTRING?
What is the reason for using macros in this case?

Reply:
use of macro is encouraged and yes, LELSTRING probably